packages/logger/src/generators/winston/generator.spec.ts

```diff
@@ -26,29 +46,95 @@ describe('logger generator', () => {
         await generator(tree, {
             ...options,
             directory: 'custom',
-            tags: 'test, client',
+            tags: 'test, logger',
         });
 
-        const config = readProjectConfiguration(tree, 'custom-test-client');
+        const config = readProjectConfiguration(tree, 'custom-test-logger');
 
         expect(config).toBeDefined();
-        expect(config.tags).toEqual(['test', 'client']);
-        expect(tree.exists('test-client/src/index.ts'));
-        expect(tree.exists('test-client/src/index.test.ts'));
+        expect(config.tags).toEqual(['test', 'logger']);
+        expect(tree.exists(`${appName}/src/index.ts`));
+        expect(tree.exists(`${appName}/src/index.test.ts`));
```

Comment:
Could be added to the below checkFilesExistInTree list?

Reply:
Moved this into part of snapshotFiles check